iot-hub/Samples/service/DigitalTwinClientSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -15,6 +14,11 @@ namespace Microsoft.Azure.Devices.Samples
 {
     public class TemperatureControllerSample
     {
+        private const string Thermostat1Component = "thermostat1";
+
+        private static readonly Uri DeviceTemperatureControllerSampleUri = 
```

Comment:
why this?

Reply:
This Uri is used in multiple methods in this file:
InvokeRebootCommandAsync
InvokeGetMaxMinReportCommandAsync

It's being used to warn the user to run the device sample before running the service sample. Created this field here to ensure that if the location of the url changed for instance, it's only updated once.

Reply:
Will change to string. Also renaming.